src/patternfly/components/DualListSelector/dual-list-selector.scss

```diff
@@ -219,6 +220,15 @@ $pf-c-dual-list-selector__item--MaxNesting: 10;
     }
   }
 
+  &.pf-m-disabled {
+    .pf-c-dual-list-selector__item-text {
+      --pf-c-dual-list-selector__item-text--Color: var(--pf-c-dual-list-selector__item--m-disabled__text--Color);
+    }
+
+    pointer-events: none;
+    cursor: none;
```

Comment:
If you add `pointer-events: none;`, you shouldn't need to style the `cursor`

Reply:
I copied this from src/patternfly/components/DataList/data-list.scss - we can fix this there as well then.